Make compatibile with latest SDK (VRCSDK3-AVATAR-2020.07.23.21.16_Public)

Latest SDK is incompatible:

```
(Filename: Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs Line: 83)

Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs(209,65): error CS0246: The type or namespace name 'VRCAvatarParameterDriver' could not be found (are you missing a using directive or an assembly reference?)
 
(Filename: Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs Line: 209)

Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs(306,66): error CS0246: The type or namespace name 'VRCAvatarParameterDriver' could not be found (are you missing a using directive or an assembly reference?)
 
(Filename: Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs Line: 306)

Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs(315,67): error CS0246: The type or namespace name 'VRCAvatarParameterDriver' could not be found (are you missing a using directive or an assembly reference?)
 
(Filename: Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs Line: 315)

Assets\Hai\ComboGesture\Scripts\Editor\Internal\ComboGestureCompilerInternal.cs(374,68): error CS0246: The type or namespace name 'VRCAnimatorTrackingControl' could not be found (are you missing a using directive or an assembly reference?)
```

--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,5 @@ VRCSDK.ShaderStripping.csproj
 Assembly-CSharp.csproj
 Assembly-CSharp-Editor.csproj
 cfe-project.sln.DotSettings.user
+VRC.SDKBase.Editor.BuildPipeline.csproj
+VRC.SDKBase.Editor.ShaderStripping.csproj

--- a/Assets/Hai/ComboGesture/Scripts/Editor/Internal/ComboGestureCompilerInternal.cs
+++ b/Assets/Hai/ComboGesture/Scripts/Editor/Internal/ComboGestureCompilerInternal.cs
@@ -5,7 +5,7 @@ using Hai.ComboGesture.Scripts.Components;
 using UnityEditor;
 using UnityEditor.Animations;
 using UnityEngine;
-using VRC.SDK3.Components;
+using VRC.SDK3.Avatars.Components;
 using VRC.SDKBase;
 using Object = UnityEngine.Object;
 

